### PR TITLE
bug 1717627. bump crv to 4.1.2

### PR DESCRIPTION
**Pull request by jcantrill**
This PR bumps CRV to 4.1.2 and adds art.yml file for future automation

### CODE REVIEW

File: manifests/4.1/cluster-logging.v4.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ metadata:
     containerImage: quay.io/openshift/origin-cluster-logging-operator:latest
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
-    olm.skipRange: ">=4.1.0 <4.1.1"
+    olm.skipRange: ">=4.1.1 <4.1.2"
```
Comment (jwforres) [inline manifests/4.1/cluster-logging.v4.1.0.clusterserviceversion.yaml]: dont touch the first one, it needs to stay at 4.1.0

File: manifests/art.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+updates:
+  - file: "{MAJOR}.{MINOR}/cluster-logging.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
+    update_list:
+    # replace metadata.name value
+    - search: "clusterlogging.v{MAJOR}.{MINOR}.0"
```
Comment (jwforres) [inline manifests/art.yaml]: this replace won't work since your metadata.name is currently 4.1.2 not 4.1.0

**Comment by jwforres** [comment]
/lgtm

approved for 4.1.2

**Comment by sudhaponnaganti** [comment]
@sosiouxme - I moved bug to POST and adding cherry-pick-approved. Is that ok?

**Pull request merged**